packages/material-ui/src/Slide/Slide.js

```diff
@@ -226,6 +227,10 @@ Slide.propTypes = {
    * A single child content element.
    */
   children: elementAcceptingRef,
+  /**
+   * If defined, then element slides in from the edge of the parent element, else it slides from the edge of the screen.
+   */
+  parent: elementAcceptingRef,
```

Comment:
This validator is used for React elements not DOM elements.

```suggestion
  parent: propTypes.object,
```

Reply:
Oh, thank you!